docs/pages/kubernetes-access/helm/reference/teleport-cluster.mdx

```diff
@@ -113,6 +113,17 @@ This field allows you to forbid specific device models and vendors, while allowi
 Devices within this list will be rejected during registration.
 By default no devices are forbidden.
 
+## `disableLocalAuthentication`
```

Comment:
Is the name a requirement?
I usually try to avoid a double negative
In this case we have `disable` and a boolean, so we might have (it's actually the default):
`disableLocalAuthentication: false`

I think we should instead have a `localAuthentication` as setting name, and a default to `true`

Reply:
I agree that a double negative is a bad idea.

Reply:
+1

Reply:
I talked about this with Hugo and suggested to use "disable" notation here because of the potential issue with how Helm treats boolean fields. It looked like we can't easily distinguish between "false" and empty value which means that "localAuth: false" and absence of "localAuth" would result in a different behavior which is a little confusing. So in the config template we won't really know whether to set local auth to "false" or not include it (to get default behavior).

Hence, the suggestion to use the "disableLocalAuth". I agree double negation is not ideal though. Maybe it's not such a big deal since we control our Helm chart and will just have this value always set in it with explicit value.

Reply:
Yep - we can set `enableLocalAuth: true` in the default `values.yaml` which will be included with all installs and someone can override it to `enableLocalAuth: false` to get this behaviour. I don't really like this either (surprise 😅) but it's definitely a lesser evil.

Reply:
We control the default value, so I think we can use `localAuth` and set the appropriate default

Reply:
I removed the double negation and the chart now always sets the setting, the executable's default have no impact.

https://github.com/gravitational/teleport/pull/13827/commits/7051356984a68c95782d28672073dc5f61d99ca1#diff-fa006d6bafda53ef7d3c1ac89b897e70648bf64a0f3f5fc5489250e0c1dc5e7dR21-R29

Does this looks OK to you?